docs/release_notes.rst

```diff
@@ -14,78 +18,4 @@ Release Notes
         _release_notes_all
 
 
-
-.. release-notes::
-   :branch: 1.0
-   :earliest-version: v1.0.0rc1
-
-.. release-notes::
-   :branch: 0.59
-   :earliest-version: v0.59.0
-
-.. release-notes::
-   :branch: 0.58
-   :earliest-version: v0.58.0
-
-.. release-notes::
-   :branch: 0.57
-   :earliest-version: v0.57.0
-
-.. release-notes::
-   :branch: 0.56
-   :earliest-version: v0.56.0
-
-.. release-notes::
-   :branch: 0.55
-   :earliest-version: v0.55.0
-
-.. release-notes::
-   :branch: 0.54
-   :earliest-version: v0.54.0
-
-.. release-notes::
-   :branch: 0.53
-   :earliest-version: v0.53.0
-
-.. release-notes::
-   :branch: 0.52
-   :earliest-version: v0.52.0
-
-.. release-notes::
-   :branch: 0.51
-   :earliest-version: v0.51.0
-
-.. release-notes::
-   :branch: 0.50
-   :earliest-version: v0.50.0
-
-.. release-notes::
-   :branch: 0.49
-   :earliest-version: v0.49.0
-
-.. release-notes::
-   :branch: 0.48
-   :earliest-version: v0.48.0
-
-.. release-notes::
-   :branch: 0.47
-   :earliest-version: v0.47.0
-
-.. release-notes::
-   :branch: 0.46
-   :earliest-version: v0.46.0
-
-.. release-notes::
-   :branch: 0.45
-   :earliest-version: v0.45.0
-
-.. release-notes::
-   :branch: 0.44
-   :earliest-version: v0.44.0
-
-
-Prior Releases
```

Comment:
Just wanna call out that we're deleting this in case it wasn't intentional, it has been long enough probably but who knows 🤷

Reply:
unintentional, good catch